### PR TITLE
feat(apps/interface): change icon svg and color

### PR DESCRIPTION
**Pull request by asaadam**
change icon svg and color

image

image

### CODE REVIEW

File: apps/interface/components/Icons/ChainBSC.tsx [hunk 1]
```diff
@@ -1,12 +1,19 @@
-import { Icon, IconProps } from "@chakra-ui/react";
+import { Icon, IconProps, useColorModeValue } from "@chakra-ui/react";
 
 export const ChainBSCIcon = (props: IconProps) => {
+    const colorChain = useColorModeValue(
```
Comment (pyk) [inline apps/interface/components/Icons/ChainBSC.tsx:4]: @asaadam We don't set the color here coz in some cases the icon requires different color. See product icon for example
Screen Shot 2022-08-09 at 13 31 49

File: apps/interface/components/Icons/ChainArbitrum.tsx [hunk 2]
```diff
@@ -1,32 +1,18 @@
-import { Icon, IconProps } from "@chakra-ui/react";
+import { Icon, IconProps, useColorModeValue } from "@chakra-ui/react";
 
 export const ChainArbitrumIcon = (props: IconProps) => {
+    const colorChain = useColorModeValue(
```
Comment (pyk) [inline apps/interface/components/Icons/ChainArbitrum.tsx:4]: @asaadam We don't set the color here coz in some cases the icon requires different color. See product icon for example
Screen Shot 2022-08-09 at 13 31 49

File: apps/interface/components/Icons/ChainArbitrum.tsx [hunk 3]
```diff
@@ -1,32 +1,18 @@
...
-            />
-            <path
-                d="M16.2 5.74928C16.1779 5.20257 15.8788 4.70209 15.4103 4.41071L8.84348 0.673336C8.38003 0.442394 7.80109 0.442098 7.33681 0.673188C7.28193 0.700587 0.950762 4.33445 0.950762 4.33445C0.863137 4.37605 0.778734 4.42558 0.699351 4.48182C0.281119 4.77846 0.0261863 5.23946 0 5.74591V13.4859L1.12818 11.7704L1.11833 5.79822C1.11964 5.77638 1.12242 5.75488 1.12635 5.73367C1.15186 5.59316 1.23113 5.4678 1.35049 5.38309C1.38027 5.36199 7.81959 1.67318 7.84005 1.66306C7.9914 1.58784 8.18377 1.58695 8.3353 1.66073L14.8171 5.35057C14.9705 5.44723 15.0666 5.6111 15.0752 5.79025V13.1883C15.0673 13.3452 15.0046 13.4906 14.8814 13.5897L13.6248 14.3103L12.9763 14.6822L10.6588 16.0115L8.30848 17.3594C8.18189 17.4047 8.03388 17.4021 7.90834 17.3516L5.12757 15.7688L4.55953 16.7222L7.0585 18.1462C7.14115 18.1927 7.21476 18.2339 7.27519 18.2675C7.36873 18.3194 7.43249 18.3541 7.45501 18.3649C7.63262 18.4503 7.88819 18.5 8.11847 18.5C8.32961 18.5 8.5355 18.4616 8.73031 18.3861L15.5569 14.4735C15.9487 14.1731 16.1792 13.72 16.2 13.2291V5.74928Z"
-                fill="#96BEDC"
-            />
+        <Icon
+            fill={colorChain}
```
Comment (pyk) [inline apps/interface/components/Icons/ChainArbitrum.tsx:10]: set fill to `currentColor`

File: apps/interface/components/Icons/ChainBSC.tsx [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
+    );
+
     return (
-        <Icon data-testid="ChainIconBSC" viewBox="0 0 16 16" {...props}>
-            <path
-                fill="currentColor"
-                d="M9.13325 12.7585V14.3733L7.72528 15.2008L6.35735 14.3733V12.7585L7.72528 13.586L9.13325 12.7585ZM1.59961 7.17335L2.96754 8.00078V10.77L5.32306 12.178V13.7928L1.59961 11.5974V7.17335ZM13.8509 7.17335V11.6041L10.0875 13.7995V12.1847L12.443 10.7767V8.00078L13.8509 7.17335ZM10.0808 4.97798L11.4888 5.80541V7.42024L9.13325 8.82821V11.6442L7.76532 12.4716L6.39738 11.6442V8.82821L3.95512 7.42024V5.80541L5.36309 4.97798L7.71861 6.38595L10.0808 4.97798ZM3.9618 8.58132L5.32973 9.40875V11.0236L3.9618 10.1961V8.58132ZM11.4888 8.58132V10.1961L10.1208 11.0236V9.40875L11.4888 8.58132ZM2.96754 3.57001L4.37551 4.39744L2.96754 5.22488V6.83971L1.59961 6.01227V4.39744L2.96754 3.57001ZM12.483 3.57001L13.891 4.39744V6.01227L12.483 6.83971V5.22488L11.1151 4.39744L12.483 3.57001ZM7.72528 3.57001L9.13325 4.39744L7.72528 5.23155L6.35735 4.40412L7.72528 3.57001ZM7.72528 0.800781L11.4888 2.99615L10.1208 3.82358L7.76532 2.41561L5.36309 3.82358L4.00183 2.99615L7.72528 0.800781Z"
-            />
+        <Icon
+            fill={colorChain}
```
Comment (pyk) [inline apps/interface/components/Icons/ChainBSC.tsx:11]: remove fill and set fill to `path` and use `currentColor`

**Review by pyk — CHANGES_REQUESTED** [review]

File: apps/interface/components/Icons/ChainBSC.tsx (hunk 1 above)
Comment (asaadam) [reply apps/interface/components/Icons/ChainBSC.tsx:4]: ah i see, okay ser

**Comment by asaadam** [comment]
@pyk do you know why the pipeline it's failing it's also happened on my another PR

**Comment by pyk** [comment]
The BNBIcon is become breaking

Screen Shot 2022-08-10 at 11 56 03

File: apps/interface/components/ChainSwitcher/index.tsx [hunk 5]
```diff
@@ -20,6 +20,7 @@ import {
 import ChainIcon from "@/components/Icons/Chain";
 import BlueIndicatorIcon from "@/components/Icons/BlueIndicator";
 import { getBaseConfig } from "@/utils/getBaseConfig";
+import { chain } from "wagmi";
```
Comment (pyk) [inline apps/interface/components/ChainSwitcher/index.tsx:23]: Use `chainSlug` from `getBaseConfig` not from `wagmi`

**Review by pyk — CHANGES_REQUESTED** [review]

**Review by pyk — APPROVED** [review]
Thanks @asaadam

**Pull request merged**